### ISSUE TITLE
Axe accessibility tests failing

### ISSUE THREAD

**Issue by bess** [projectblacklight/blacklight#2493]
The Axe accessibility tests are failing in CI so they have been disabled temporarily. 

Failures here: 

```
Selenium::WebDriver::Error::JavascriptError:
       javascript error: unknown rule `` in options.rules
       JavaScript stack:
       Error: unknown rule `` in options.rules
           at eval (eval at executeScript (:445:16), <anonymous>:14:338341)
           at Array.forEach (<anonymous>)
           at Au.value (eval at executeScript (:445:16), <anonymous>:14:338297)
           at Au.value (eval at executeScript (:445:16), <anonymous>:14:336077)
           at eval (eval at executeScript (:445:16), <anonymous>:14:350451)
           at new Promise (<anonymous>)
           at Object.axe.runPartial (eval at executeScript (:445:16), <anonymous>:14:350414)
           at eval (eval at executeAsyncScript (:521:26), <anonymous>:7:36)
           at eval (eval at executeAsyncScript (:521:26), <anonymous>:19:4)
           at executeAsyncScript (<anonymous>:521:47)
         (Session info: headless chrome=94.0.4606.81)
```

![Screen Shot 2021-10-18 at 3 23 34 PM](https://user-images.githubusercontent.com/65608/137794020-930100ec-d61c-4e40-9ec4-ce2dc4fd704c.png)
